salt/states/pkg.py

```diff
@@ -508,8 +508,11 @@ def _find_install_targets(name=None,
         # add it to the kwargs.
         kwargs['refresh'] = refresh
 
+
+    resolve_capabilities = kwargs.get('resolve_capabilities', False) and 'pkg.list_provides' in __salt__
```

Comment:
This is not entirely correct. `pkg.list_provides` will be always available (unless I am missing some magic here) in Zypper module. So why checking? Besides `__salt__['pkg.list_provides'](...)` is the same as just `list_provides(...)` from within here. Normally you supposed to call stuff via `__salt__` if this is about to other modules around, so you reuse `__virtual__`. But if you are within the same module, you just call things directly.

Reply:
We are here in the package state and not in the zypper module

Reply:
See above.